index.html

```diff
@@ -14,29 +14,22 @@ <h2 class="page_title">TODO list</h2>
                     New task:
                 </td>
                 <td>
-                    <form action="">
-                        <input type="text" name="" id="" placeholder="Insert your task here">
-                        <input type="submit" value="CREATE" class="button">
-                    </form>
+                        <input type="text" name="" id="" placeholder="Insert your task here" class="i-1">
+                        <button type="" id="create_task">CREATE</button>
                  </td>
             </tr>
         </table>
         <br>
         <span>My tasks:</span>
         <hr>
-        <div>
-            <ul>
-                <li>
-                    піся
-                </li>
-                <li>
-                    кака
-                </li>
-                <li>
-                    попа
-                </li>
+        <div class="out">Here
+            <ul id="ul">
```

Comment:
It is best to give ids specific names. What does this element hold?
Probably this id makes more sense: taskList

Reply:
Renamed on optimal name